src/test/scala/uk/gov/hmrc/perftests/example/MarginalReliefCalculatorRequestsRequests.scala

```diff
@@ -118,5 +118,31 @@ object MarginalReliefCalculatorRequestsRequests extends ServicesConfiguration {
       .get(s"$baseUrl$route/results-page": String)
       .check(status.is(200))
 
+  val fullResultsPage: HttpRequestBuilder =
+    http("Get full results page")
+      .get(s"$baseUrl$route/full-results-page": String)
+      .check(status.is(200))
+
+  val PDFMetaDataPage: HttpRequestBuilder =
+    http("Get pdf meta data page")
+      .get(s"$baseUrl$route/pdf-meta-data": String)
+      .check(status.is(200))
+      .check(saveCsrfToken)
+
+  val postPDFMetaDataPage: HttpRequestBuilder =
+    http("Post pdf meta data page")
+      .post(s"$baseUrl$route/pdf-meta-data": String)
+      .formParam("Company name (optional)", "abcdefg")
+      .formParam("UTR number (optional)", "abcdefg")
+      .formParam("csrfToken", s"$${csrfToken}")
+      .check(status.is(303))
+      .check(header("Location").is(s"$route/pdf").saveAs("pdf"))
+
+  val PDFPage: HttpRequestBuilder =
```

Comment:
can we use the correct casing for the variable and also rename to savePDF or downloadPDF?